src/commands/api-mesh/log-list.js

```diff
@@ -0,0 +1,110 @@
+const { Command } = require('@oclif/core');
+
+const logger = require('../../classes/logger');
+const { initSdk, initRequestId } = require('../../helpers');
+const { ignoreCacheFlag, jsonFlag, fileNameFlag } = require('../../utils');
+const { getMeshId, listLogs } = require('../../lib/devConsole');
+const { appendFileSync, existsSync } = require('fs');
+const { ux } = require('@oclif/core/lib/cli-ux');
+const path = require('path');
+
+require('dotenv').config();
+class ListLogsCommand extends Command {
+	//static args = [{ name: 'file' }];
+	static flags = {
+		ignoreCache: ignoreCacheFlag,
+		json: jsonFlag,
```

Comment:
@dani97 what does this jsonFlag do? write it in json format in the csv file?

Reply:
Going to remove this, I took reference from other commands not needed here

Reply:
they use it for more verbose output in json. here we wont need it